elastic/kibana
# [dashboard] fix 'Each child in list should have a unique key prop' warning in PresentationPanelHeader

Fixes
<img width="600" alt="Screenshot 2024-06-27 at 2 47 07 PM" src="https://github.com/elastic/kibana/assets/373691/44d0162c-ae6f-4995-a780-711ff59727a8">


### test steps
* in cloud 7.17. instance, create legacy input control visualization and export saved object
* in local instance, import legacy input control vis
* create new dashboard and add legacy input control vis
* Open console and ensure no warnings are displayed when `deprecated` badge is displayed
